docs/meetings/collab-cafe/2024.md

```diff
@@ -1,5 +1,73 @@
 # 2024 Collaboration Cafe Notes Archive
 
+## 2024-02-20
+
+### Check-in :raising_hand:
+
+- Min / @minrk / Simula
+- Sarah / @sgibson91 / 2i2c
+- Samuel / @sgaist / Idiap Research Institute
+- Kirstie / @KirstieJane / The Alan Turing Institute
+- Simon / @manics / University of Dundee
+- Steve / @stevejpurves / Curvenote | MystMarkdown
+
+### Introduce yourself! :wave:
+
+_If you are new to the meeting, add your name below and you can introduce yourself at the start of the meeting._
+
+- Kirstie Whitaker - Programme Director for Tools, Practices and Systems at The Alan Turing Institute. Founder of The Turing Way. Very interested in open source governance (EDI, accountability, transparency, project management etc)
+
+### Agenda :clock2:
+
+- **Kirstie** (15mins): CZI Essential Open Source Software for Science (EOSS) Diversity, Equity and Inclusion (DEI) funding revamp proposal
+  - <https://docs.google.com/document/d/1pSU9kG_XDOSTF4JcbIt18hep1rCHIjuUYuYi5mg0q-0/edit>
```

Comment:
```suggestion
  - <https://docs.google.com/document/d/1pSU9kG_XDOSTF4JcbIt18hep1rCHIjuUYuYi5mg0q-0/edit?usp=sharing>
```

I changed the setting on the doc so its now suggestion only (I hope?)

Thank you for catching @manics !

Reply:
Yep, it's suggestion only when I visit it!